src/solver/ts-generator/xcast/xcast.cpp

```diff
@@ -58,11 +58,6 @@ XCast::XCast(Data::Study& study, Data::TimeSeriesType ts, IResultWriter& writer)
 {
 }
 
-XCast::~XCast()
```

Comment:
About (**XCast**'s constructor) : 
```c++
    pNeverInitialized(true),
    pAccuracyOnCorrelation(false),
```
We should prefer initializing these data members inside class declaration

Reply:
Agreed, will do